pkg/controller/status/status.go

```diff
@@ -496,6 +540,14 @@ func (m *statusManager) syncState() {
 		}
 	}
 
+	for _, w := range m.windowsnodeupgrades {
```

Comment:
the syncState function this is in will be called every 5 seconds, if there are lots of nodes being upgraded all at once we'll be spamming the apiserver with Get'ing node. I guess this will only be a concern when upgrading from OS -> Ent.
Maybe we should not have the status manager do a separate Get of the Node and let the WindowsUpdate/Reconcile update `m.windowsnodeupgrades` and the status manager can use that state for status information. WDYT?

Reply:
This is a good suggestion, addressed in c5fa45f3498b1adc2e8fcc363728fe2bcf755cf1